tests/test_language_metadata.py

```
@@ -0,0 +1,523 @@
+    collection = _FakeChromaCollection()
+    backend = MultiRootBackend.__new__(MultiRootBackend)
+    backend._cols = [("docs", None, collection)]
+
+    result = backend.query([0.1], k=1, language_filter="es")
+
+    assert result["ids"] == [["doc-1"]]
+    assert collection.query_kwargs["where"] == {"language": "es"}
+
+
+class _FakeZvecDoc:
+    def __init__(self, id="doc-1", score=-0.2, **fields):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)